### ISSUE TITLE
Hardcoded variables

### ISSUE THREAD

**Issue by richardsimongreen** [moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate#2]
The step asks for variables to be given but inside it seems to use predefined named variables:
-u $SAUCELABS_USERNAME  -k $SAUCELABS_ACCESS_KEY

If you make these the names of your secrets and then add them then it works and is a workaround.
If not then it is just broken

**Comment by moatazeldebsy** [comment]
Yes they should be added as secrets to be able to use them with bitrise

**Comment by richardsimongreen** [comment]
Ah ok thx, probably should not be fields to enter then

**Comment by sergavt** [comment]
this time it added the REDACTED in the echo, so it definitely picked up those vars, but same error is returned: cannot execute binary file
![Screenshot 2023-05-11 at 11 18 01 AM](https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate/assets/93228869/e906303c-92fc-4cfe-a14a-6aa8f28c45f4)
